content/60.methods.md

```diff
@@ -2,179 +2,264 @@
-Media outlets don't always report on this accurately.
-Additionally, many papers are being retracted.
-These include both preprints and papers that are published in more traditional venues.
-The large number of retractions may also be influenced by the fact that the time from submission to peer review for papers related to COVID-19 is very low.
-
-The rate of this proliferation also presents challenges to efforts to summarize and synthesize existing literature, which are necessary given the volume.
-A number of groups have sought to track and review COVID-19 preprints<!--To Do: add refrences-->.
-However, any static review is likely to quickly become dated as new research is released or existing research is retracted or superseded, and the explosive rate of publication made localized efforts to curate new publications increasingly difficult.
-Additionally, the complex nature of COVID-19 means that significant advantages can be gained from examining the virus and disease in a multidisciplinary context.
+The downsides of "excessive publication" have been recognized for over forty years, as it was raised as a major concern about the move towards electronic, rather than print, publishing at the turn of the millennium [@doi:10/d3bmnv].
+The contents of the COVID-19 Open Research Dataset (CORD-19) [@arxiv:2004.10706], which was developed in part to assist in efforts to train machine learning algorithms on COVID-19-related text, illustrates the volume of publication relevant to understanding this virus (Figure @fig:cord19-growth).
+This resource was developed by querying several sources for terms related to SARS-CoV-2 and COVID-19, as well as the coronaviruses SARS-CoV-1 and MERS-CoV and their associated viruses [@arxiv:2004.10706] and contained {{cord19_total_pubs}} manuscripts as of {{cord19_date_pretty}}.
+Additional curation by CoronaCentral [@doi:10.1101/2020.12.21.423860] has produced, at present, a set of over 150,000 publications particularly relevant to COVID-19 and these closely related viruses.
+Thus, any effort to synthesize, summarize, and contextualize COVID-19 research will face a vast corpus of potentially relevant material.
+
+**Change over time in the number of publications in the CORD-19 dataset.**
+As of {{cord19_date_pretty}}, there are {{cord19_total_pubs}} in the CORD-19 dataset.
+The first release, on March 16, 2020, contained 28,000 manuscripts on topics relevant to SARS-CoV-2 and related coronaviruses [@arxiv:2004.10706].
+Since then, these articles have continued to proliferate (left), with both traditionally published and preprint manuscripts in the corpus (right).
+At present, it contains {{cord19_total_preprints}} preprints from _arXiv_, _bioRxiv_, and _medRxiv_.
+While not all of the manuscripts are focused explicitly on SARS-CoV-2 or COVID-19, this corpus is likely to contain all or most manuscripts relevant to writing a literature review, which requires assessing both emerging and prior research.
```

Comment:
In the future, one could imagine examining the fraction of papers that we cite that are in CORD.